rpc/server_auth.go

```diff
@@ -162,6 +162,12 @@ func (ss *simpleServer) signAccessTokenForEntity(
 	return tokenString, nil
 }
 
+func (ss *simpleServer) canByPassAuthCheck(
```

Comment:
```suggestion
func (ss *simpleServer) isPublicMethod(
```
[opt]
if we are going to keep this function, `canBypassAuthCheck` seems like a complex name for this simple function

Reply:
changed to IsPublicMethod